### ISSUE TITLE
default interaction mode disable all default events

### ISSUE THREAD

**Issue by dberardo-com** [lukasbach/react-complex-tree#7]
with reference: https://rct.lukasbach.com/docs/guides/interaction-modes

**Describe the bug**
custom interaction mode disable all default events

**To Reproduce**
include the following prop to a ControlledComponent

```
 defaultInteractionMode={{
          mode: "custom",
          createInteractiveElementProps: (
            item,
            treeId,
            actions,
            renderFlags
          ) => ({
            onDoubleClick: () => {
              actions.focusItem();
              actions.selectItem();
              treeRef?.current?.startRenamingItem(item.index);
            },
          }),
        }}
```

I can achieve the intended behavior, i.e. that the list item goes into renaming mode on doubleClik, but all other events are not handled anymore, like onClick, etc.

**Comment by lukasbach** [comment]
This is now fixed as of version v1.1.3 of this package. Please update your dependency to receive the fix.

You can now provide an `extends` prop to your interaction manager, that specifies which existing interaction manager is used as base for all events. More details are available at https://rct.lukasbach.com/docs/guides/interaction-modes#custom-interaction-modes

If this issue or a similar problem persists, feel free to reopen this issue or create a new one. Thanks again for your issue report! :)

**Comment by dberardo-com** [comment]
I have noticed that now that i override the onClick and onDoubleClick hanlders, i cannot manage to let the onDoubleClick event fire anymore, since the onClick is always fired instead. is this an actual issue or is it only on my side?
